Controllers/OrdersController.cs

```diff
@@ -37,14 +39,18 @@ public async Task<ActionResult<IEnumerable<GetDataDTO<OrdersDTO>>>> GetOrders()
 
 
         [Authorize(Roles = "Warehouse")]
-        [HttpGet("{id}")]
-        public async Task<ActionResult<GetDataDTO<OrdersDTO>>> GetOrdersByWarehouse(int id)
+        [HttpGet("byID")]
+        public async Task<ActionResult<GetDataDTO<OrdersDTO>>> GetOrdersByWarehouse()
```

Comment:
This is essentially the same method as `GetOrders()`. Maybe we could use smart filtering there instead similarly as it is done in `ProductBalancesController.GetProductBalances()` method?

Reply:
This is wouldn't the same method because taking data is different.